### DISCUSSION TITLE
tutorials/lift-off-part4/02-what-is-a-mutation

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#54] [course feedback]
# tutorials/lift-off-part4/02-what-is-a-mutation

https://www.apollographql.com/tutorials/lift-off-part4/02-what-is-a-mutation

**Comment by attilacsanyi** [comment]
Nice, I learned new best practice here which is to return common Response shape for mutations :thanks

**Comment by nwazuo** [comment]
OH my! I love these 'fill-in-the-blanks' exercise variants. They help me verify that I have the correct words in my head about a concept.

**Comment by le-el** [comment]
I love this course

**Comment by omriluz** [comment]
Great Course!, really interactive and I am learning a lot while enjoying it

**Comment by YuliaCoolinich** [comment]
"Correct. Queries and mutations are essential to GraphQL. In a future course, we'll cover the last type of GraphQL operation: subscriptions."
Could you please help to find this course part with subscriptions?

**Reply by mabuyo** [reply]
Hey there @YuliaCoolinich! We don't have a course on subscriptions available yet, unfortunately. I did find a couple other resources on subscriptions that could be helpful:

- [Docs](https://www.apollographql.com/docs/react/data/subscriptions/) (Apollo Client web)
- [Summit 2023 Workshop Slides for Federated Subscriptions in GraphOS](https://drive.google.com/file/d/1DWU2IOfp3dRPIoUDiw-tmh1voD4KpFYJ/view?usp=sharing)

What are you looking to learn with subscriptions? Happy to take note of your feedback for future courses.
